requirements.txt

```diff
@@ -0,0 +1,28 @@
+atomicwrites==1.1.5
+attrs==18.1.0
+certifi==2018.8.13
+chardet==3.0.4
+click==6.7
+colorama==0.3.9
+coverage==4.5.1
+coveralls==1.3.0
+docopt==0.6.2
+Flask==1.0.2
+Flask-API==1.0
+gunicorn==19.9.0
+idna==2.7
+itsdangerous==0.24
+Jinja2==2.10
+MarkupSafe==1.0
+more-itertools==4.3.0
+nose==1.3.7
+pipenv==2018.7.1
+pluggy==0.7.1
+py==1.5.4
+pytest==3.7.2
+requests==2.19.1
+six==1.11.0
+urllib3==1.23
+virtualenv==16.0.0
```

Comment:
Why is virtual env in your requirements. Its better for it to be a installed on your global interpretter. You maybe working on a project where people create virtual environments differently forexample using virtualenvwrapper

Reply:
> Preview

emoved virtual environment